tools/licenses/lib/licenses.dart

```diff
@@ -338,6 +344,11 @@ abstract class License implements Comparable<License> {
       return true;
     }());
     final LicenseType detectedType = convertBodyToType(body);
+
+    // Fuchsia SDK Vulkan license is Apache 2.0 with some additional BSD-matching copyrights.
+    if (type == LicenseType.vulkan)
+      yesWeKnowWhatItLooksLikeButItIsNot = true;
```

Comment:
Alternatively, I could add && `type != LicenseType.vulkan` in the two conditionals right below this, but in fairness this _is_ what this var is for -- despite not usually being a fan of mutating function params.